libvast/src/system/meta_index.cpp

```diff
@@ -17,6 +17,7 @@
 #include "vast/expression.hpp"
 #include "vast/fbs/utils.hpp"
 #include "vast/logger.hpp"
+#include "vast/partition_synopsis.hpp"
```

Comment:
nit: I think it's preferable to skip includes that are already pulled in the header of the current source file.